extensions/blocks/simple-payments/edit.js

```diff
@@ -513,7 +513,6 @@ class SimplePaymentsEdit extends Component {
 						className={ classNames( 'simple-payments__field', 'simple-payments__field-title', {
 							'simple-payments__field-has-error': fieldTitleError,
 						} ) }
-						label={ __( 'Item name', 'jetpack' ) }
```

Comment:
This may be here for screen readers (even if we are hiding it in the layout).

Reply:
could we use `aria-label` for this? Instead of using a hacky display none solution?

Reply:
> could we use `aria-label` for this? Instead of using a hacky display none solution?

It looks like aria-label can be used ["when the label of the control is clear from the surrounding content."](https://www.w3.org/WAI/tutorials/forms/labels/#using-aria-label)

Reply:
I've removed the `display: none` since it isn't recommended when we want to guarantee a11y. Instead of this, I've implemented some CSS rules in order to handle it properly. Some doc about this [here](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/). Also, we did a [similar approach for the podcast player](https://github.com/Automattic/jetpack/blob/master/extensions/blocks/podcast-player/style.scss#L25).
CLosing it as resolved.